doc/releases/changelog-dev.md

```diff
@@ -407,6 +407,52 @@
 
 <h3>Improvements</h3>
 
+* `qml.dot` now groups coefficients together.
+  [(#3691)](https://github.com/PennyLaneAI/pennylane/pull/3691)
+
+  ```pycon
+  >>> qml.dot(coeffs=[2, 2, 2], ops=[qml.PauliX(0), qml.PauliY(1), qml.PauliZ(2)])
+  2*(PauliX(wires=[0]) + PauliY(wires=[1]) + PauliZ(wires=[2]))
+  ```
+
+* `qml.generator` now supports `Sum` and `Prod` instances.
```

Comment:
Makes it sound like `qml.generator(Sum)` is supported, not that `qml.generator(op, format="prefactor") returns sum or product instances.

Reply:
Changed!